docs/1.getting-started/3.configuration.md

```diff
@@ -210,4 +210,8 @@ export default defineNuxtConfig({
 })
 ```
 
+#### experimental `reactivityTransform` migration from Vue 3.4 and Nuxt 3.9
+
+Since Nuxt 3.9 and Vue 3.4, `reactivityTransform` has been  moved from Vue to Vue Macros which has a [Nuxt integration](https://vue-macros.dev/guide/nuxt-integration.html).
```

Comment:
```suggestion
Since Nuxt 3.9 and Vue 3.4, `reactivityTransform` has been moved from Vue to Vue Macros which has a [Nuxt integration](https://vue-macros.dev/guide/nuxt-integration.html).
```